### DISCUSSION TITLE
Histogram improvements

### DISCUSSION THREAD

**Discussion by GWeidmann** [datalust/seq-tickets#1977] [Ideas]
I like the new appearance of the histogram, but it's missing two small features that would really be a benefit to me:

![image](https://github.com/datalust/seq-tickets/assets/97590580/4e17e85e-8f55-47e4-9c69-59f2d4e56792)

* Total count of found results
* Let me choose a timespan (1m, 5m, 1h, ...) by myself

**Comment by nblumhardt** [comment]
Thanks for the suggestion! We're exploring some possibilities in this space currently, it's good timing 👍

**Reply by GWeidmann** [reply]
Hello, any update on this?

**Reply by nblumhardt** [reply]
Thanks for the nudge, @GWeidmann 👍 

We've just run through the current state of the histogram; we're pretty sold on:

 - [ ] Making the time grouping interval selectable (`15m`, `1h`, etc.)
 - [ ] Showing tool tips and permitting click interactions across the whole height of each bar, not just the (potentially small) click target at the bottom
 - [ ] Always showing the full width of the final bar, even when it's only partially filled
 - [ ] Setting the current date range with click-and-drag, rather than just clicks on single bars

The loading indicator should be fixed as of the recent Seq versions. We're not sure about adding the full count, at this stage.

I'll create issues tracking the list above where they're missing, not sure it'll make 2026.1, but having an issue rather than a discussion should get things some more attention :-)

Cheers!

**Reply by nblumhardt** [reply]
https://github.com/datalust/seq-tickets/issues?q=is%3Aissue%20state%3Aopen%20frequency%20histogram

**Reply by GWeidmann** [reply]
Sounds great, thanks a lot for the fast reply :-)

**Comment by GWeidmann** [comment]
One addition: If I switch the time span e.g. from `1d` to `7d`, it can take a while to load. But at the moment no loading indicator is visible. This works great with the dashboards though.
